4.x/typescript/Recenter/README.md

```diff
@@ -0,0 +1,273 @@
+# Recenter custom widget App
+
+This tutorial goes over creating a custom widget that displays the [MapView.center](https://developers.arcgis.com/javascript/latest/api-reference/esri-views-MapView.html#center)'s X/Y coordinates in addition to its [MapView.scale](https://developers.arcgis.com/javascript/latest/api-reference/esri-views-MapView.html#scale). In addition to displaying coordinates and scale, the view can also be recentered by clicking on the widget.
+
```

Comment:
Add a note about how to install and serve up the fully completed sample in this repo, that's also where they can find the actual source code for the snippet.

Reply:
You might want to include a minimum requirements section up front. They'll need to pre-install TypeScript, NPM, etc to get these steps to work as stand-alone tasks. The full source code does have a package.json file which installs dependencies, but they won't have the benefit of it in these steps,

Reply:
> Add a note about how to install and serve up the fully completed sample in this repo, that's also where they can find the actual source code for the snippet.

I don't know if we should be providing one folder to just click and get it started. We don't do this in the SDK samples either. IMO, it's easier to provide the links to topics showing how to get started in TS and the pre-requisites needed in order to get a new TS project up and running to be able to do the steps. I updated the Readme to point to either the demo app in this repo or to the custom hello world widget in the SDK which was recently updated as well.

I added a new folder for source files, this includes the index.html file, .tsx, and compiled .js file.

Reply:
> I don't know if we should be providing one folder to just click and get it started. We don't do this in the SDK samples either.

Yah, that's how we handle it under /esm-samples. There's no tutorials, just install and go.